src/internal/m365/collection/drive/permission.go

```diff
@@ -390,7 +390,12 @@ func UpdateLinkShares(
 		}
 
 		if err != nil {
-			el.AddRecoverable(ctx, clues.Stack(err))
+			if graph.IsErrSharingDisabled(err) {
```

Comment:
nit: These could be added as an `else ifs` at line 390.

Reply:
imo, else-if chains are difficult to read.  However, they could be unnested, since `graph.IsErrFoo` will return falsy for any nil error.  Ex:

```go
if graph.IsErrSharingDisabled(err) {
  logit...
  continue
}

if err != nil {
  addRecoverable
  continue
}
```

Imo, that's the easiest format to read.